packages/kbn-search-connectors/types/native_connectors.ts

```diff
@@ -315,6 +321,56 @@ export const NATIVE_CONNECTOR_DEFINITIONS: Record<string, NativeConnector | unde
         validations: [],
         value: '',
       },
+      data_center_username: {
+        default_value: null,
+        depends_on: [
+          {
+            field: 'data_source',
+            value: 'confluence_data_center',
+          },
+        ],
+        display: DisplayType.TEXTBOX,
+        label: i18n.translate(
+          'searchConnectors.nativeConnectors.confluenceDataCenter.usernameLabel',
+          {
+            defaultMessage: 'Confluence Data Center username',
```

Comment:
This should just be 'username', we don't want to prefix this stuff with the application name -- users already know they're looking at a confluence data center config.

Reply:
I've removed prefix as a product name 👍